Cargo.toml

```diff
@@ -8,6 +8,7 @@ license = "MIT"
 readme = "README.md"
 
 [dependencies]
+cargo = "0.23"
```

Comment:
I'd rather be independent from the cargo crate. Can we just use `String` instead of `SourceId` for now?